rm/rm-node/build.gradle

```diff
@@ -20,10 +20,10 @@ dependencies {
     runtime "org.objectweb.proactive:programming-extension-pamr:${programmingVersion}"
 
     // script engines
-    runtime 'org.jruby:jruby-complete:9.0.5.0'
-    runtime 'org.python:jython-standalone:2.7.0'
+    runtime 'org.jruby:jruby-complete:9.1.12.0'
+    runtime 'org.python:jython-standalone:2.7.1'
```

Comment:
I looked through the jython news and could find contradictory information: 
1) Jython news: https://hg.python.org/jython/rev/0df7adb1b397 (2.7.1 release is a proper release)
2) Maven Repository: https://mvnrepository.com/artifact/org.python/jython (2.7.1b3 is the latest) 

I think it is safe to use 2.7.1 but mvnrepository.com doesn't seem to be up to date.